ui/src/app/components/help-sidebar-icons.tsx

```diff
@@ -0,0 +1,666 @@
+import * as React from 'react';
```

Comment:
Most of this file is migrated from `help-sidebar`.  I'll point out notable changes.